ddtrace/appsec/processor.py

```diff
@@ -89,15 +89,22 @@ class _Addresses(object):
     "x-real-ip",
 }
 
-_COLLECTED_HEADER_PREFIX = "http.request.headers."
 
-
-def _set_headers(span, headers):
-    # type: (Span, Dict[str, Union[str, List[str]]]) -> None
+def _set_headers(span, kind, headers, extra_collect={}):
+    # type: (Span, str, Dict[str, Union[str, List[str]]], Set[str]) -> None
+    always_collect = frozenset(
```

Comment:
maybe put that outside the function as a global/constant?

Reply:
Done